recipes/install_binary.rb

```diff
@@ -27,7 +27,7 @@
   version node['consul_template']['version']
   checksum install_checksum
   url ::URI.join(node['consul_template']['base_url'],
-                 "v#{node['consul_template']['version']}/",
+                 "#{node['consul_template']['version']}/",
```

Comment:
URLs have dropped the 'v' since the move, eg https://releases.hashicorp.com/consul-template/0.12.0/